guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -33,12 +33,26 @@ include::../common/modules/snip_warning-maintain-config-noop.adoc[]
 # {foreman-installer} --foreman-proxy-dns-managed=false \
 --foreman-proxy-dhcp-managed=false
 ----
-ifdef::katello[]
-. Optional: If you use PostgreSQL as an external database, install the `postgresql-evr` package on the PostgreSQL server:
```

Comment:
I am re-purposing the `postgresql-evr` section here, as there is *no way* users can have arrived here without having `postgresql-evr` installed on their SQL servers.

Reply:
and removing it for 3.8+ in https://github.com/theforeman/foreman-documentation/pull/2579